mathics/builtin/vectors/vector_space_operations.py

```diff
@@ -160,6 +164,12 @@ def eval(self, u: ListExpression, v: ListExpression, evaluation):
 
 class UnitVector(Builtin):
     """
+    <url>
+    :Unit vector:
+    https://en.wikipedia.org/wiki/Unit_vector</url> (<url>
+    :WMA:
+    https://reference.wolfram.com/language/ref/UnitVector.html</url>)
```

Comment:
This is split into multiple lines without the "\", isn't this a problem?

Reply:
I just copy and paste another existing case...

Reply:
> This is split into multiple lines without the backslash, isn't this a problem?

The home-grown code is copying the nature of HTML and XML (where XML is a more fancy worLd for HTML because XML schemas are not supported; the HTML schema for XML is assumed. So while you will see references to XML in the code, HTML is meant.).

In HTML, recall that newlines are not significant. Recall you'd use `<br/>` to force a new line, or maybe a `<p>` would do that as well. 

The problem with the homegrown approach is, well, that it is _homegrown_. I imagine there are XML/HTML Python libraries out there, but this homegrown code isn't using any of those. 

As with most homegrown code,  it is in incomplete in various ways or has bugs.

(I can't help be reminded between parallels with the flaws of the homegrown HTML code and homegrown code to to change it. Please, let us learn from this mistake. Or, if not that, our constant mistake that whenever homegrown code is written, accept that is going to have bugs. And when code you know is going to have bugs, don't try to shove the whole codebase through it to start out.)

 Back to the homegrown doc system. In particular, it does not reflow paragraphs the way a HTML renderer typically does. When it sees newlines in non-tagged text, it just copies that to output.

Why don't we fix this? Well, the code is a mess and the thing should have never been used in the first place. We should have hooked into a Python doctest system, since that was intended.  My own feeling is that I'd rather put effort into ditching this for something more powerful and that is maintained by others in standard package like sphinx than fixing this homegrown system. 

So sadly, we've been limping along with this crap.  

The homegrown doctest system does however strip extra spaces between words.  And that is the feature we are making use of with adding `\` at the end of a line. Since this inside a Python triple-quoted string --  _Python_ is turning the "\n" into " ".

Reply:
In any case, it seems to work for now. Something that I have back in my mind when I wrote the automatic replacement was to see if it would be possible to write an automatic translation from our homebrew documentation system to some more standard like sphinx (as @rocky suggested several times) without the need of rewriting al the documentation by hand. In any case, even adding the URL field for the proper Builtin subclasses is not an easy task...